tests/BigIntegerPrimeNumberTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace PHP\Math\BigIntegerTest;
+
+use PHP\Math\BigInteger\BigInteger;
+use PHPUnit_Framework_TestCase;
+
+class BigIntegerPrimeNumberTest extends PHPUnit_Framework_TestCase
+{
+    public function testBigIntegerIsPrimeNumber()
+    {
+        // not prime number definitely
+        $bigInteger = new BigInteger('6');
+
+        $notPrimeNumber = $bigInteger->isPrimeNumber();
+
+        // a prime number probably
+        $bigInteger = new BigInteger('1111111111111111111');
+
+        $isProbPrimeNumber = $bigInteger->isPrimeNumber();
+
+        // a prime number definitely
+        $bigInteger = new BigInteger('11');
+
+        $isPrimeNumber = $bigInteger->isPrimeNumber();
+
+        // Assert
+        $this->assertTrue($isProbPrimeNumber);
```

Comment:
Could you split this up into three separate tests? There is too much happening in the unit test.

I suggest:
- testIsPrimeNumberWithtPrimeNumber()
- testIsPrimeNumberWithoutPrimeNumber()
- testIsPrimeNumberWithoutProbabilePrimeNumber()

Reply:
Also try to follow the AAA pattern:
- Arrange
- Act
- Assert

I split them up via comments:
```
// Arrange

// Act

// Assert
```

Reply:
Also, the assert methods are static so you should call them like: static::assertTrue :)